src/app/config.ts

```diff
@@ -1,15 +1,17 @@
-function appConfig () {
-   
-    return {
-        base_url: 'http://127.0.0.1:8000/api/v1/'
-    };
+function appConfig() {
+  return {
+    base_url: 'https://landville-backend-web-api.herokuapp.com/api/v1/',
+    facebookId: '1782833315349267',
+    googleId:
```

Comment:
@kelvinndmo, Could there be any catch including the google id and the facebook app id in version control?
If that's the case then we could consider using environment variables.